bugsnag-android-core/src/test/java/com/bugsnag/android/StacktraceSerializationTest.kt

```diff
@@ -69,7 +69,7 @@ internal class StacktraceSerializationTest {
                     }
                 }
             }
-            return Stacktrace(elements)
+            return Stacktrace(elements.toMutableList())
```

Comment:
This can be done better in the above `(0..999).map` by changing it to `mapTo(ArrayList())`